src/utils/rootkeypackage_utils/src/rootkeypackage_do_download.cpp

```diff
@@ -20,7 +20,7 @@ ADUC_Result DownloadRootKeyPkg_DO(const char* url, const char* targetFilePath)
     Log_Info("Downloading File '%s' to '%s'", url, targetFilePath);
 
     const std::error_code ret = microsoft::deliveryoptimization::download::download_url_to_path(url, targetFilePath);
-    if (ret == std::error_code())
+    if (! ret)
```

Comment:
should it be doing ret.value() explicit == check with 0 ?

Reply:
I was following the standard here:

``` const std::error_code doErrorCode = MSDO::download::download_url_to_path(
    entity->DownloadUri, fullFilePath.str(), false, std::chrono::seconds(retryTimeout));
    if (!doErrorCode)
    {
        resultCode = ADUC_Result_Download_Success;
        extendedResultCode = 0;
    }
```

I can explicitly check ret.value() == 0 if that is easier.